src/relay/transforms/pattern_utils.h

```diff
@@ -450,6 +450,23 @@ static inline Array<Integer> ToVector(const runtime::NDArray& array) {
   return out;
 }
 
+/*!
+ * \brief Convert a NDArray with type int or float to Array<FloatImm>.
+ * \param array Input NDArray
+ * \return Converted Array.
+ */
+static inline Array<FloatImm> ToFloatVector(const runtime::NDArray& array) {
```

Comment:
Just curious, why was this needed? I dont see how it interacts with resize.

Reply:
I used it in DynamicToStatic to convert the ROI argument.